galaxy/api/internal/serializers/collections.py

```diff
@@ -112,3 +112,10 @@ def get_all_versions(self, obj):
             hidden=False
         )
         return VersionListSerializer(versions, many=True).data
+
+
+class CollectionUpdateSerializer(serializers.ModelSerializer):
```

Comment:
Optional: From REST API design standpoint for partial updates `PATH` is preferred over `PUT`. `PUT` method is usually used for sending full object state.